funnel/models/proposal.py

```diff
@@ -354,6 +305,26 @@ def roles_for(self, actor=None, anchors=()):
             roles.remove('reader')  # https://github.com/hasgeek/funnel/pull/220#discussion_r168724439
         return roles
 
+    def assign_label(self, label):
```

Comment:
IMO, this method should be on the Label class as `apply_to(proposal)`.

Reply:
The reason being that we treat functionality in layers. Proposals are a base layer of functionality and labels are added on top. Lower layers should not be dependent on upper layers. If we remove some functionality in future, it should be as simple as removing the db model and the corresponding code module.

Reply:
We also need a corresponding `remove_from(proposal)`.